### ISSUE TITLE
Language toggle is not in a landmark

### ISSUE THREAD

**Issue by benoitlord** [benoitlord/portfolio_nuxt#9]
### Issue Description
Vérifier que tout le contenu est localisé dans une région

### Element location
`.language-toggle-container`

### Element source
`<a data-v-2938d5c6="" href="/fr/portfolio/fast" class="language-toggle-container change" lang="fr" aria-label="Changer au français">`

### To solve this violation, you need to...
Fix the following:
- Contenu non imbriqué dans une région ARIA
